source/common/json/json_sanitizer.cc

```diff
@@ -65,7 +65,7 @@ absl::string_view sanitize(std::string& buffer, absl::string_view str) {
   if (need_slow == 0) {
     return str; // Fast path, should be executed most of the time.
   }
-  TRY_ASSERT_MAIN_THREAD {
+  try {
```

Comment:
why this change? It will fail lint (as it should).

Reply:
we will construct json log line at worker threads, so this change is necessary.

Reply:
I think a separate change is needed to solve this problem. This won't pass lint; we can't catch exceptions in worker threads.

Reply:
How? Make this exception free directly? My initial plan was to create an exception free version sanitize method. But you hope to defer that to future and take much time to convince me.